src/nav/style-guide.yml

```diff
@@ -58,6 +58,8 @@ pages:
         path: /docs/style-guide/writing-guidelines/formatting-terminal-commands
   - title: Processes and procedures
     pages:
+      - title: Tech writer workflow
```

Comment:
This is called `Tech writer workflow` here, but `Tech Writer Workflow` in the actual page frontmatter.